
Remove recent xterm terminfo features to be compatible with other emulations

--- a/misc/terminfo.src	2021-03-20 22:45:39.000000000 +0100
+++ b/misc/terminfo.src	2021-06-01 23:53:10.335516419 +0200
@@ -4808,8 +4808,8 @@ xterm-xfree86|xterm terminal emulator (X
 xterm+nofkeys|building block for xterm fkey-variants,
 	npc,
 	kcbt=\E[Z, kent=\EOM, nel=\EE, use=ecma+index,
-	use=ansi+rep, use=ecma+strikeout, use=vt420+lrmm,
-	use=xterm+sm+1006, use=xterm+tmux, use=ecma+italics,
+	use=ecma+strikeout, use=vt420+lrmm,
+	use=xterm+sm+1005, use=xterm+tmux, use=ecma+italics,
 	use=xterm+keypad, use=xterm-basic,
 
 # This version reflects the current xterm features.
